### DISCUSSION TITLE
Lesson 8: error RPC Anvil chain -  Add to Metamask

### DISCUSSION THREAD

**Discussion by Perso77** [Cyfrin/foundry-full-course-cu#100] [Q&A] [answered]
why this rpc not available?

![metamask](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/eed3b0e6-4b5c-44d5-b53f-c6f98122c946)

**Comment by ghost** [comment, marked as answer]
Hello @Perso77 this issue usually occurs if you try to add the network but you don't have the local chain running. Run `anvil` in your terminal and try again.

**Reply by cromewar** [reply]
This is most likely the solution, @Perso77 does this make sense for you?

**Reply by Perso77** [reply]
Yes sir.
Thanks @pacelliv 

![METAMASK1](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/db1d7f26-f0e8-4b4a-ae60-a769777c2945)
